src/main/java/bisq/desktop/main/account/content/fiataccounts/FiatAccountsView.java

```diff
@@ -17,93 +17,39 @@
 
 package bisq.desktop.main.account.content.fiataccounts;
 
+import bisq.common.UserThread;
+import bisq.common.util.Tuple2;
+import bisq.common.util.Tuple3;
+import bisq.core.app.BisqEnvironment;
+import bisq.core.locale.Res;
+import bisq.core.payment.*;
+import bisq.core.payment.payload.PaymentMethod;
+import bisq.core.util.validation.InputValidator;
 import bisq.desktop.common.view.ActivatableViewAndModel;
 import bisq.desktop.common.view.FxmlView;
 import bisq.desktop.components.AutoTooltipButton;
 import bisq.desktop.components.AutoTooltipLabel;
 import bisq.desktop.components.TitledGroupBg;
-import bisq.desktop.components.paymentmethods.AliPayForm;
-import bisq.desktop.components.paymentmethods.CashAppForm;
-import bisq.desktop.components.paymentmethods.CashDepositForm;
-import bisq.desktop.components.paymentmethods.ChaseQuickPayForm;
-import bisq.desktop.components.paymentmethods.ClearXchangeForm;
-import bisq.desktop.components.paymentmethods.FasterPaymentsForm;
-import bisq.desktop.components.paymentmethods.InteracETransferForm;
-import bisq.desktop.components.paymentmethods.MoneyBeamForm;
-import bisq.desktop.components.paymentmethods.NationalBankForm;
-import bisq.desktop.components.paymentmethods.OKPayForm;
-import bisq.desktop.components.paymentmethods.PaymentMethodForm;
-import bisq.desktop.components.paymentmethods.PerfectMoneyForm;
-import bisq.desktop.components.paymentmethods.PopmoneyForm;
-import bisq.desktop.components.paymentmethods.RevolutForm;
-import bisq.desktop.components.paymentmethods.SameBankForm;
-import bisq.desktop.components.paymentmethods.SepaForm;
-import bisq.desktop.components.paymentmethods.SepaInstantForm;
-import bisq.desktop.components.paymentmethods.SpecificBankForm;
-import bisq.desktop.components.paymentmethods.SwishForm;
-import bisq.desktop.components.paymentmethods.USPostalMoneyOrderForm;
-import bisq.desktop.components.paymentmethods.UpholdForm;
-import bisq.desktop.components.paymentmethods.VenmoForm;
-import bisq.desktop.components.paymentmethods.WesternUnionForm;
+import bisq.desktop.components.paymentmethods.*;
```

Comment:
Could you please use imports like @cbeams mentioned in https://github.com/bisq-network/bisq-core/pull/70#pullrequestreview-110353024. I've fixed the imports in my local repo already. Could you please check "[Allow edits from maintainers](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/)" so I can push my local changes to your fork?

Reply:
Already kicked 'Allow edits from maintenaners' please check if it is ok.

Just curious, are you doing this by manually update import and not select the 'optimize import' during commit?

Because I had tried again in my workspace, no matter using auto import or optimize import during commit, it would update into .* style..

Reply:
Worked for me thanks! I'm not committing directly from Intellij, but from and external tool (just my personal habit), so I guess that's why the `optimize import` never occurred to me. But good to know if I change my worfklow.

As soon as https://github.com/bisq-network/bisq-core/pull/70#pullrequestreview-110520877 is changed this PR is ACK from my side. Did one trade on RegTest with the changes.

Reply:
For your next contributions, please use signed commits as explained here: https://help.github.com/articles/signing-commits-using-gpg/. Then you'll have the nice `Verified` label next to your commits 😄